storage/s3storage/option.go

```diff
@@ -70,3 +71,24 @@ func WithExpiration(exp time.Duration) Option {
 		}
 	}
 }
+
+func WithFileStorageClass(storageClass string) Option {
+	return func(h *S3Storage) {
+		switch storageClass {
```

Comment:
Seems clearer and simpler keeping the string as it is?
i.e. `REDUCED_REDUNDANCY`, `STANDARD_IA` etc

Reply:
Done.